ui/components/Notification/MesheryNotification.js

```diff
@@ -0,0 +1,182 @@
+import react, {useState} from "react";
+import { styled } from "@mui/material/styles";
+import {
+    IconButton,
+    Typography,
+    Divider,
+    Tooltip,
+    Badge,
+    Drawer,
+    Tabs,
+    Tab,
+  } from "@mui/material";
+  import NotificationsIcon from '@mui/icons-material/Notifications';
+  import DoneAllIcon from '@mui/icons-material/DoneAll';
+  import MesheryEventViewer from "./MesheryEventViewer";
+
+   
+  function MesheryNotification () {
+
+  const DrawerButton = styled(IconButton)((theme) => ({
```

Comment:
Before creating a new button, ask yourself whether this button can be derived from an existing one or whether this button has any similarities with the existing ones. Try to reuse components and styles as much as possible.